pkg/operator/ceph/cluster/osd/health.go

```diff
@@ -184,3 +213,29 @@ func (m *OSDHealthMonitor) restartOSDIfStuck(osdID int) error {
 	}
 	return nil
 }
+
+// updateCephStorage udates the CR with deviceclass details
+func (m *OSDHealthMonitor) updateCephStorage(devices []string) {
```

Comment:
Please add a unit test for this method. See other unit tests in health_test.go for the osd health monitor.

Reply:
Unit test has been added